aim/web/ui/src/pages/Experiment/useExperimentData.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { IResourceState } from 'modules/core/utils/createResource';
+import { IExperimentData } from 'modules/core/api/experimentsApi';
+
+import experimentEngine from './ExperimentStore';
+
+function useExperimentData(experimentId: string) {
+  const { current: engine } = React.useRef(experimentEngine);
+  const experimentStore: IResourceState<IExperimentData> =
+    engine.experimentState((state) => state);
+  const experimentsStore: IResourceState<IExperimentData[]> =
+    engine.experimentsState((state) => state);
+
+  React.useEffect(() => {
+    return () => {
+      engine.destroyExperiment();
+      engine.destroyExperiments();
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  React.useEffect(() => {
+    engine.fetchExperimentData(experimentId as any);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [experimentId]);
+
+  return {
+    experimentStore,
```

Comment:
What is the benefit of changing the name of `experiment(s)State` to `experiment(s)Store`?
As for the names `State` and `Stores`, I believe the engine should export the `Store` and the hook should export `State`. 
`State` - since the state, by the context of reactive programming is considered Observable by default
`Store` - since the store isn't observable in nature, it has some methods to get/subscribe to the Observable state and at the same time store has a lot of other functions, you can not render something directly from the store, I mean if we want to call some variable `State`, it should be considered as data, but the store isn't data at all.

i.e. I believe you don't use name Store as expected